crates/orchestrator/src/tests/server/mod.rs

```diff
@@ -11,40 +13,51 @@ use starknet::providers::JsonRpcClient;
 use url::Url;
 use utils::env_utils::{get_env_var_or_default, get_env_var_or_panic};
 
+use crate::config::Config;
 use crate::queue::init_consumers;
-use crate::routes::app_routes::app_router;
-use crate::tests::config::TestConfigBuilder;
+use crate::routes::app_routes::{app_router, handler_404};
+use crate::routes::job_routes::job_routes;
+use crate::tests::config::{ConfigType, TestConfigBuilder};
 
 #[fixture]
-pub async fn setup_app_server() -> SocketAddr {
+pub async fn setup_server() -> (SocketAddr, Arc<Config>) {
     dotenvy::from_filename("../.env.test").expect("Failed to load the .env.test file");
 
     let madara_url = get_env_var_or_panic("MADARA_RPC_URL");
     let provider = JsonRpcClient::new(HttpTransport::new(
         Url::parse(madara_url.as_str().to_string().as_str()).expect("Failed to parse URL"),
     ));
 
-    TestConfigBuilder::new().configure_starknet_client(provider.into()).build().await;
+    let services = TestConfigBuilder::new()
```

Comment:
the `setup_server` should take `services` as an argument because the health server doesn't need it for example

Reply:
resolved with new tokio server spin up